orchestration/aws/infra.py

```diff
@@ -50,7 +50,7 @@ def connect_to_queue(self, queue_name: str):
         self.queue = boto_resource.get_queue_by_name(QueueName=queue_name)
         return True  # TODO Is this needed?
 
-    def listen(self, callback: callable[[dict[str, Any]], None]):
+    def listen(self, callback: Callable[[dict[str, Any]], None]):
```

Comment:
🤦🏻‍♀️ good catch, I got so high on getting rid of Typing with generics that I went too far.